### PR TITLE
Pd 331 rbac

### PR DESCRIPTION
**Pull request by nathan-px**
I just took a stab at this based on what I saw in the ticket.

### CODE REVIEW

File: content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* A running Portworx cluster on Kubernetes with role-based access control
+
+## Modify a cluster role
+
+1. As a Kubernetes cluster administrator, create a new Kubernetes cluster role, or find an existing role you wish to modify.
+
+2. Modify the role, adding the Stork rules to it:
+
+
+        apiVersion: rbac.authorization.k8s.io/v1
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md]: This should be in a code box?

File: content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+description: Learn how to configure role-based access control to allow your users to perform Stork operations.
+noicon: true
+weight: 11
+---
+
+Cluster administrators can allow certain users to perform Stork operations by creating and assigning custom roles for their users. Under the configuration described below, a Cluster administrator defines permissions in a user role which allows the user to perform Stork operations.
+
+## Prerequisites
+
+* A running Portworx cluster on Kubernetes with role-based access control
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md]: Don't need to mention this. The role can be added even if there is no PX or stork

File: content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Modify a cluster role
+
+1. As a Kubernetes cluster administrator, create a new Kubernetes cluster role, or find an existing role you wish to modify.
+
+2. Modify the role, adding the Stork rules to it:
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md]:
```suggestion
2. Modify the role, adding permissions for Stork resources to it:
```

File: content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Configure role-based access control for Stork"
+linkTitle: "Configure access control for Stork"
+keywords: cloud, backup, restore, snapshot, DR, migration, kubemotion
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md]:
```suggestion
keywords: cloud, backup, restore, snapshot, DR, migration, kubemotion, stork, rbac, role, clusterrole
```

File: content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+weight: 11
+---
+
+Cluster administrators can allow certain users to perform Stork operations by creating and assigning custom roles for their users. Under the configuration described below, a Cluster administrator defines permissions in a user role which allows the user to perform Stork operations.
+
+## Prerequisites
+
+* A running Portworx cluster on Kubernetes with role-based access control
+
+## Modify a cluster role
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md]:
```suggestion
## Modify a Kubernetes role
```
or 
```suggestion
## Modify a role
```

Role and ClusterRole are 2 different types of objects

File: content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Configure role-based access control for Stork"
+linkTitle: "Configure access control for Stork"
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md]:
```suggestion
linkTitle: "Configure role-based access control for Stork"
```

File: content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md (hunk 1 above)
Comment (nathan-px) [reply content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md]: In this case, no. The `-` character doesn't work with the traditional code box syntax, this level of indentation is an alternative way of creating a code box.

File: content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+description: Learn how to configure role-based access control to allow your users to perform Stork operations.
+noicon: true
+weight: 11
+---
+
+Cluster administrators can allow certain users to perform Stork operations by creating and assigning custom roles for their users. Under the configuration described below, a Cluster administrator defines permissions in a user role which allows the user to perform Stork operations.
+
+## Modify a role
+
+1. As a Kubernetes cluster administrator, create a new Kubernetes cluster role, or find an existing role you wish to modify.
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md]:
```suggestion
1. As a Kubernetes cluster administrator, create a new Kubernetes role, or find an existing one you wish to modify.
```

Comment (disrani-px) [reply content/portworx-install-with-kubernetes/storage-operations/rbac-stork/_index.md]: `Role` is an object type, so maybe you want to highlight that?

**Pull request merged**